### PR TITLE
Fixes menu

### PR DESCRIPTION
**Pull request by kul3r4**
Solves #751 
@sebastianbenz PTAL

### CODE REVIEW

File: src/50_Samples_%26_Templates/Product.html [hunk 1]
```diff
@@ -260,13 +260,13 @@
       <!-- ## Navigation -->
       <!-- Use the [amp-sidebar](/components/amp-sidebar/) component to give customers the chance to quickly jump to any of your product categories.-->
       <amp-sidebar id="drawermenu" layout="nodisplay">
-        <div class="topheader">
-          <a href="/" class="home">Products by Example</a>
+        <div class="topheader m1">
```
Comment (sebastianbenz) [inline src/50_Samples_%26_Templates/Product.html]: is topheader still needed? I think we can get rid of the div and add "block p1" to the link instead.

File: src/50_Samples_%26_Templates/Product.html [hunk 2]
```diff
@@ -260,13 +260,13 @@
...
+          <a href="/" class="caps text-decoration-none">Products by Example</a>
         </div>
         <hr/>
-        <h4 class="category expanded"><a href="/samples_templates/product_listing/preview/">Fruit</a></h4>
-        <h4 class="category"><a href="/samples_templates/product_listing/preview/">Vegetable</a></h4>
-        <h4 class="category"><a href="/samples_templates/product_listing/preview/">More</a></h4>
+        <h4><a class="m1 caps text-decoration-none" href="/samples_templates/product_listing/preview/">Fruit</a></h4>
```
Comment (sebastianbenz) [inline src/50_Samples_%26_Templates/Product.html]: we could get rid of the `<h4>` tag and use the class `h4` instead

Comment (kul3r4) [inline src/50_Samples_%26_Templates/Product.html]: This will make all the links in the same line.
Would it be ok to add a <br/> after each one?

Comment (sebastianbenz) [inline src/50_Samples_%26_Templates/Product.html]: Same as above, add `class"block p1"`

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

**Comment by sebastianbenz** [comment]
The header has default styling, but each link is styled as `h4`. Suggestion: use h4 for heading and no default styling for links. 

Can you make the same changes to the product listing sample?

**Comment by kul3r4** [comment]
Applied the latest changes, thanks!

**Pull request merged**